---
home: true
footer: Â© Park Lab. All rights reserved.
---

## Data Science for Epigenetics

<img src="/img/top.jpg">

### Our Mission

<p>Our central research interest is to understand tissue heterogeneity, stem cell homeostasis, and cellular aging using genomics, epigenetics, and advanced informatics. We develop informatics for DNA methylation assays, including Infinium DNA methylation microarray, bisulfite-sequencing, and single-molecule, long-read sequencing-based technologies. We are dedicated to providing sensible interpretation of various forms of epigenetic data and their integration. We create statistical inference methods to query epigenetic cell identity, cancer cell of origin, evolution, and immune microenvironment.</p>

### Our Research

#### Investigating large-scale behavior in human diseases using bioinformatics

<div class="home">
  <dl>
    <dt><img src="/img/pic1.png"></dt>
    <dd>We believe DNA methylation is one of the best markers for cell identity due to its biochemical stability and allelic presence in most cells. We measure and analyze DNA methylation signals to investigate tissue and tumor heterogeneity, tumor cell-of-origin, immune microenvironment for personalized cancer medicine, and immune therapy. We develop computational methods to perform linear deconvolution of bulk tissue DNA methylation signal. </dd>
  </dl>
</div>

#### Identifying molecular mechanisms of human diseases across genetic and epigenetic data

<div class="home">
  <dl>
    <dt><img src="/img/pic2.png"></dt>
    <dd>Global DNA methylation loss is a signature of the cancer epigenome. It occurs primarily within lamina-associated, late-replicating regions termed partially methylated domains (PMDs). Our analysis revealed previously undetected PMD hypomethylation in virtually all healthy tissues. PMD hypo-methylation increased with age, beginning during fetal development, and appeared to track the accumulation of cell divisions.</dd>
  </dl>
</div>

#### Developing computational methods to predict treatment outcomes

<div class="home">
  <dl>
    <dt><img src="/img/pic3.png"></dt>
    <dd>Illumina Infinium DNA Methylation BeadChips (HM450, EPIC) and high throughput bisulfite sequencing (BS-seq) represent the two most successful and widely used genome-scale DNA methylation platforms of assay. We develop statistical methods and software to control the quality of these experiments and new analytics so that investigators get the most out of their measurements.</dd>
  </dl>
</div>

#### Elucidating translational implications of the predictions

<div class="home">
  <dl>
    <dt><img src="/img/pic4.png"></dt>
    <dd>Global DNA methylation loss is a signature of the cancer epigenome. It occurs primarily within lamina-associated, late-replicating regions termed partially methylated domains (PMDs). Our analysis revealed previously undetected PMD hypomethylation in virtually all healthy tissues. PMD hypo-methylation increased with age, beginning during fetal development, and appeared to track the accumulation of cell divisions.</dd>
  </dl>
</div>
